IDS/Converter.php

```diff
@@ -158,7 +158,7 @@ public static function convertFromJSCharcode($value)
                 if (preg_match_all('/\d*[+-\/\* ]\d+/', $char, $matches)) {
                     $match = preg_split('/(\W?\d+)/',
                                         (implode('', $matches[0])),
-                                        null,
+                                        '',
```

Comment:
3rd param is an int for preg_split?

Reply:
Otherwise PR looks good.

Reply:
ah good point will update to -1

Reply:
ok fixed now @demeritcowboy

Reply:
Cool. It turns out -1 and 0 mean the same thing in that spot (previously null would have been cast to 0).